### DISCUSSION TITLE
Selected is being removed from Async Multi

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4865] [Q&A] [answered]
Hello, i noticed that when u select an option from the async multi it is bein removed from the list (menu) and being added to the input. Any way to keep it in the list and mark it as selected ?

**Comment by ebonow** [comment, marked as answer]
Sounds like you are looking for the boolean prop `hideSelectedOptions` which defaults to true

**Reply by cubecleveland** [reply]
superb tanks!
